app/gui2/e2e/selectingNodes.spec.ts

```diff
@@ -23,14 +30,12 @@ test('Selecting nodes by click', async ({ page }) => {
   await locate.graphNodeIcon(node1).click({ modifiers: ['Shift'] })
   await expect(node1).toBeSelected()
   await expect(node2).toBeSelected()
+  await deselectAll()
 
   await locate.graphNodeIcon(node2).click()
   await expect(node1).not.toBeSelected()
   await expect(node2).toBeSelected()
-
-  await page.mouse.click(600, 200)
-  await expect(node1).not.toBeSelected()
-  await expect(node2).not.toBeSelected()
```

Comment:
Another case removed: clicking at the background should deselect all nodes. I actually had regressions there: please restore it.